### PR TITLE
Fea 136 fix initial central place on map

### PR DESCRIPTION
**Pull request by adrian-konopnicki-CC**

### CODE REVIEW

**Review by klaudia-blazyczek-cc — APPROVED** [review]

File: src/pages/Map/MapPage.tsx [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
 import { PlaceMarkers } from './components/PlaceMarkers';
 import { StyledMapContainer, MapWrapper } from './MapPage.styled';
 import { MapPageTestIds } from './MapPageTestIds';
 
-const initialBounds = '(50.72167742756552,16.311950683593754,51.48993452350156,17.76351928710938)';
 const initialZoom = 10;
 
 const MapPage = () => {
-  const localization: [number, number] = [51.107, 17.038];
+  const [localValue] = useLocalStorage('current_location');
```
Comment (kamil-borecki-cc) [inline src/pages/Map/MapPage.tsx]: **suggestion** change name of this variable to smth more descriptive like currentLocation

File: src/pages/Map/MapPage.tsx [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
+  const location = data?.location;
+
+  const defaultPosition: [number, number] = [51.107, 17.038];
 
   return (
     <MapWrapper data-testid={MapPageTestIds.MapPage}>
       <StyledMapContainer
-        center={localization}
-        zoom={10}
+        center={location ? [location.lat, location.lon] : defaultPosition}
```
Comment (kamil-borecki-cc) [inline src/pages/Map/MapPage.tsx]: **question** it looks like the map will jump to another position after data is loaded, am i right?

File: src/pages/Map/components/PlaceMarkers/PlaceMarkers.tsx [hunk 3]
```diff
@@ -33,9 +39,16 @@ const PlaceMarkers = ({ boundsCoordinates, zoom }: Props) => {
 
   const map = useMapEvents({
     zoomend: () => setMapPositionDebounce(map),
-    dragend: () => setMapPositionDebounce(map),
+    dragend: () => {
```
Comment (kamil-borecki-cc) [inline src/pages/Map/components/PlaceMarkers/PlaceMarkers.tsx]: **question** why this change was made?

File: src/pages/Map/MapPage.tsx (hunk 1 above)
Comment (kamil-borecki-cc) [reply src/pages/Map/MapPage.tsx]: **suggestion** to avoid `jumping` map maybe better option will be to store another item in localstorage with coordinates of current location?

File: src/pages/Map/components/PlaceMarkers/PlaceMarkers.tsx (hunk 3 above)
Comment (adrian-konopnicki-CC) [reply src/pages/Map/components/PlaceMarkers/PlaceMarkers.tsx]: It's actually a leftover.

File: src/pages/Home/HomePage.tsx [hunk 4]
```diff
@@ -10,8 +10,13 @@ import { Container, SelectContainer, WidgetWrapper } from './HomePage.styled';
 import { HomePageTestIds } from './HomePageTestIds';
 
 const HomePage = () => {
+  const [chosenCity, setChosenCity] = useState('');
+
   const [savedLocation, setSavedLocation] = useLocalStorage('current_location');
```
Comment (kamil-borecki-cc) [inline src/pages/Home/HomePage.tsx]: **suggestion** unify names of thiose variables. savedLocation => currentLocation, setSavedLocation => setCurrentLocation

Comment (adrian-konopnicki-CC) [reply src/pages/Home/HomePage.tsx]: Done

**Review by AgataLudwiczynska-CC — APPROVED** [review]

**Review by darya-karpovich-cc — APPROVED** [review]

File: src/pages/Home/HomePage.tsx [hunk 5]
```diff
@@ -57,9 +62,22 @@ const HomePage = () => {
...
+    (value: string) => {
+      setChosenCity(value);
+      setCurrentLocation(value);
+    },
+    [setCurrentLocation],
+  );
+
+  const updateCoordinates = useCallback(async () => {
+    const { data } = await refetch();
+    data && setCurrentCoordinates(`[${data.location.lat}, ${data.location.lon}]`);
```
Comment (kamil-borecki-cc) [inline src/pages/Home/HomePage.tsx]: **chore** this function is redundant

current coordinates are additional data and it can be obtained when data is loaded (which was done before)

Comment (adrian-konopnicki-CC) [reply src/pages/Home/HomePage.tsx]: Yeah, you're right

**Review by kamil-borecki-cc — APPROVED** [review]

**Review by klaudia-blazyczek-cc — APPROVED** [review]

**Review by AgataLudwiczynska-CC — APPROVED** [review]
